asyncwrapper/src/main/kotlin/org/n/riesgos/asyncwrapper/datamanagement/DatamanagementRepo.kt

```diff
@@ -101,26 +102,29 @@ class DatamanagementRepo (
         var jobIdSetNotSetYet = true
         for (complexInputKey in complexInputs.keys) {
             val complexInput = complexInputs.get(complexInputKey) as ComplexInputConstraint
-            val searchForThisComplexInput = complexInputRepo.findByProcessWpsIdentifierInputWpsIdentifierLinkMimetypeXmlSchemaAndEncoding(
+            val searchForThisComplexInput = complexInputRepo.findByProcessWpsIdentifierJobStatusInputWpsIdentifierLinkMimetypeXmlSchemaAndEncoding(
```

Comment:
Best. Method-name. Ever.

Reply:
I know that it is quite verbose, but spring-data-jpa uses similar names (but those can create the method ifself just with a name like this). I wanted to be a little bit inline with the way spring-data-jpa or panache work.